cypress/support/pages/login_management/pageLoginManagement.js

```diff
@@ -8,6 +8,7 @@ class Login_Management {
   static #infoMessage = '[data-testid="info-message"]'
   static #submitButton = '[data-testid="btn-submit"]'
   static #cancelButton = '[data-testid="btn-cancel"]'
+  static #brokerButton = '[data-testid="submit-oauth-login"]'
 
   visitLoginPage() {
     cy.visit(Cypress.env('BRB'))
```

Comment:
here the cy.visit() should call the 'DEFAULT' as 'OIDCMOCK' button is available on default so it fails this test.

Reply:
Changed to DEFAULT instead of BRB